spirv_msl.cpp

```diff
@@ -27,13 +27,20 @@ using namespace std;
 
 static const uint32_t k_unknown_location = ~0;
 
-CompilerMSL::CompilerMSL(vector<uint32_t> spirv_)
+CompilerMSL::CompilerMSL(vector<uint32_t> spirv_, vector<MSLVertexAttr> *p_vtx_attrs,
+                         std::vector<MSLResourceBinding> *p_res_bindings)
```

Comment:
Removed unnecessary use of std:: in cpp file.

Incoming vectors cannot be marked as `const` because the values in the vectors may be modified during compilation to have their `used_by_shader` flag set. This is documented in constructor API.